moveit_ros/planning/planning_scene_monitor/src/trajectory_monitor.cpp

```diff
@@ -63,7 +63,7 @@ void planning_scene_monitor::TrajectoryMonitor::setSamplingFrequency(double samp
 
 bool planning_scene_monitor::TrajectoryMonitor::isActive() const
 {
-  return record_states_thread_;
+  return !!record_states_thread_.get();
```

Comment:
On the kinetic-devel branch this is currently already fixed as return static_cast<bool>(record_states_thread_); Not sure why we didn't backport it, but I think it's best to use the same fix here too.